### PR TITLE
added motivation.json

### PR DESCRIPTION
**Pull request by sarthak5**

### CODE REVIEW

File: data/motivation.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"motivation": [
```
Comment (swapagarwal) [inline data/motivation.json]: motivations

**Comment by swapagarwal** [comment]
Rename file to motivations.json

**Comment by swapagarwal** [comment]
Have you gone through all of quotes, or do you need help? Also, remove them from quotes.json

**Comment by sarthak5** [comment]
Yeah I've gone through all quotes and found these motivational. I'll remove these from quotes.json and change name to motivations.json.

**Comment by swapagarwal** [comment]
Thanks @sarthak5! 👍

**Pull request merged**